.gitattributes

```diff
@@ -2,4 +2,5 @@
 # line endings. Windows users contributing to Go will need to use a
 # modern version of git and editors capable of LF line endings.
 
-*.go -text diff=golang
\ No newline at end of file
+*.go -text diff=golang
+*.go text eol=lf
```

Comment:
you mean to check this in?

Reply:
Yes, as we have developers using Windows and this ensures that go files won't have CRLF line endings